<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>601</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_voxel_resolution">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Voxel resolution</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_voxel_resolution">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>0.001000000000000</double>
    </property>
    <property name="maximum">
     <double>0.100000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
    <property name="value">
     <double>0.005000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="button_update">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QListView" name="list_labels">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>101</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_new_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>New label</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_segment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Segment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete label</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_labels">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Labels</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_update</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonUpdateClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_new_label</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonNewLabelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_segment</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonSegmentClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_delete_label</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonDeleteLabelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonUpdateClicked()</slot>
  <slot>buttonNewLabelClicked()</slot>
  <slot>buttonSegmentClicked()</slot>
  <slot>buttonDeleteLabelClicked()</slot>
 </slots>
</ui>
